src/main/resources/application.properties

```diff
@@ -1,23 +1,30 @@
-# Configuration file
-# key = value
 quarkus.swagger-ui.path=/doc
 quarkus.swagger-ui.always-include=true
 
-#org.jboss.pnc.cleaner.orchapi.BuildRecordEndpoint/mp-rest/url=
-org.jboss.pnc.cleaner.orchapi.BuildRecordEndpoint/mp-rest/scope=javax.inject.Singleton
+# PNC endpoints
+org.jboss.pnc.cleaner.orchapi.BuildRecordEndpoint/mp-rest/url=http://localhost:8082
+org.jboss.pnc.cleaner.orchapi.BuildRecordEndpoint/mp-rest/scope=javax.enterprise.context.ApplicationScoped
 
+org.jboss.pnc.cleaner.orchapi.BuildConfigSetRecordEndpoint/mp-rest/url=http://localhost:8082
+org.jboss.pnc.cleaner.orchapi.BuildConfigSetRecordEndpoint/mp-rest/scope=javax.enterprise.context.ApplicationScoped
+
+# Failed builds cleaner
 # start at 1:15AM every day
 failedbuildscleaner.cron=0 15 1 * * ?
 failedbuildscleaner.retention=168
 failedbuildscleaner.indy.requesttimeout=30
-#failedbuildscleaner.indyurl=
-
+failedbuildscleaner.indyurl=http://localhost:8083
 
-#keycloak.refreshifexpiresinseconds=
+# Authentication
+keycloak.refreshifexpiresinseconds=3600
+serviceaccount.realm=FIXME
+serviceaccount.realmpublickey=FIXME
+serviceaccount.authserverurl=http://localhost:8084
+serviceaccount.sslrequired=FIXME
+serviceaccount.resource=FIXME
+serviceaccount.secret=FIXME
```

Comment:
I intentionally left those commented to not get any unwanted values into published binary and get an error when starting up. With these you won't get notified that you are missing some setting. Do you think this is better approach?

Reply:
I see it as better as it allow us to test it easily. We set all in one place, otherwise we would need to replace all while testing and the config would be invalid anyway and meaningless. 

In production we will replace the values, by what we need and the app won't work anyway with the empty or dummy config.

Reply:
Yes, it won't work, but you notice only when it triggers a scheduled job. But when it's missing, younnotice immediately after starting the service... Anyway, it's not a big issue. I just prefer to not have there any dummy values, becauswe they are different for every environment anyway so you need to set them everytime.

Reply:
I checked options to set it and there is not many. Maintaining tests would be much harder as we would need to set all as env values or use custom config/application.properties files for testing, which sounds like worse option. I prefer to keep the one inside the repository as a test profile.